modules/power-monitoring-dashboards-overview.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+
+// * power_monitoring/visualizing-power-monitoring-metrics.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="power-monitoring-dashboards-overview_{context}"]
+= {PM-shortname-c} dashboards overview
+
+There are two types of {PM-shortname} dashboards. Both provide different levels of details around power consumption metrics for a single cluster:
+
+Power Monitoring / Overview dashboard:: 
+With this dashboard you can observe the following information:
++
+--
+* An aggregated view of CPU architecture and its power source (`rapl-sysfs`, `rapl-msr`, or `estimator`) along with total nodes with this configuration
+
+* Total energy consumption by a cluster in the last 24 hours (measured in kilowatt-hour)
+
+* The amount of power consumed by the top 10 namespaces in a cluster in the last 24 hours
+
+* Detailed node information, its CPU architecture and component power source
+--
++
+These features allow you to effectively monitor the energy consumption of the cluster without needing to necessarily investigate every namespace separately.
++
+[WARNING]
+====
+If {PM-kepler} is not able to obtain hardware power consumption metrics, the *Components Source* column will display `estimator` as the power source, which is not supported. If that happens, then the values from the nodes are not accurate.
+====
++
+.Power Monitoring / Overview dashboard
+image::power-monitoring-overview-dashboard.png[]
+
+Power Monitoring / Namespace dashboard:: 
+This dashboard provides the possibility to view metrics by namespace and pod. You can observe the following information:
++
+--
+* The power consumption metrics, such as consumption in DRAM and PKG
+
+* The energy consumption metrics in the last hour, such as consumption in DRAM and PKG for core and uncore components
+--
++
+This feature allows you to investigate key peaks and easily identify the primary root causes of high consumption.
++
+.Power Monitoring / Namespace dashboard
+image::power-monitoring-namespace-dashboard.png[]
```

Comment:
This is something I was thinking about as well. I do not like to add images overall. But this seemed important for the engineers. I will discuss with them, if they would be okay with not adding these after all.

If they will have to stay:
* I tried to put the images in the beginning and it looks much worse actually :grinning: so I will leave it in the end. The other thing is, there is not much description to be added there, because they show what is explained in the text above them. So I was thinking that adding this type of text:

The following image shows the described features:

Or something  similar to that, would be the best. But first I will try to see if the images can be removed. Thank you!